ADOPTERS.md

```diff
@@ -0,0 +1,122 @@
+# Adopters
```

Comment:
Is the file duplicated here, and in `docs`? I think one is enough on the website, or we can just symlink it.

Reply:
Good call, it's doable and I will do it.

Reply:
Aaand it's done.